apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerTransactionCreationOptionsTests.cs

```diff
@@ -150,6 +152,24 @@ public void WithIsDetached()
         Assert.False(options.IsDetached);
     }
 
+    [Fact]
+    public void WithReadLockMode()
```

Comment:
You need to check that the default is unspecified on a few of the tests above this line.